sherpa/models/basic.py

```diff
@@ -1843,7 +1843,8 @@ def __init__(self, name='usermodel', pars=None):
             pars = (self.ampl,)
         else:
             for par in pars:
-                self.__dict__[par.name] = par
```

Comment:
This line has not been changed since we created the initial git repository. Any idea what introduced the regression?

Reply:
As I mentioned in the commit notes; #255 changed the logic for the __getattr__/__setattr__ behavior, adding in information to the model when a parameter was added. This code was not being called here, when editing __dict__ directly. Or at least that's my belief.

Reply:
[I probably should have added the last sentence as a separate comment rather than when I was editing it because of markdown]

Reply:
Thanks.

For the record, I don't think git bisect requires all the tests to pass at every commit (or even the code to build at every commit) and one can add skipping conditions for compilation errors and other unrelated issues, even when using `git bisect run` to automate the search rather than running interactively. I don't think it matters in this case anyway.